src/components/sortable_table/SortableTable.js

```diff
@@ -11,6 +11,34 @@ import { ThemeProvider } from '@mui/material/styles';
 import EnhancedTableHead from './EnhancedTableHead';
 import { theme } from './Themes';
 
+// headCells and rows should be in the following format:
+
+// const headCells = [
+//   {
+//     id: 'id1',
+//     label: 'label1', // this will be displayed as the column name
+//   },
+//   {
+//     id: 'id2',
+//     label: 'label2',
+//   }
+// ];
+
+// const rows = [
+//   {
+//     id1: 'id1_value1', // use the id values from headCells for each item
+//     id2: 'id2_value1'
+//   },
+//   {
+//     id1: 'id1_value2',
+//     id2: 'id2_value2'
+//   },
+//   {
+//     id1: 'id1_value3',
+//     id2: 'id2_value3'
+//   }
+// ];
```

Comment:
nit: rm commented lines

Reply:
Sure! My original intention for including it was to have a guide for people who want to use SortableTable in the future. But I can remove those lines if it's better.

Reply:
Ah, I didn't get that. There is a standard for JS doc ... it would be good to use that at the top of the component. Look at `BucketUtils.js` for an example.